src/styles/index.scss

```diff
@@ -1,3 +1,95 @@
+@import "./values.scss";
+@import "./card.scss";
+@import "./stars.scss";
+
 body {
   margin: 0;
 }
+
+.text-main {
+  color: $Main-accent;
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 18px;
+}
+
+.text-secondary {
+  color: $Secondary;
+  font-family: "Roboto", sans-serif;
```

Comment:
No need to duplicate font-family everywhere, you can add font family to html selector